linodecli/plugins/obj.py

```diff
@@ -275,13 +275,27 @@ def _do_multipart_upload(bucket, filename, file_path, file_size, policy):
 
     print("{} ({} parts)".format(filename, num_chunks))
 
+    num_tries = 3
+    retry_delay = 2
+
     try:
         with open(file_path, "rb") as f:
             for i in range(num_chunks):
                 print(" Part {}".format(i + 1))
-                upload.upload_part_from_file(
-                    f, i + 1, cb=_progress, num_cb=100, size=MULTIPART_UPLOAD_CHUNK_SIZE
-                )
+                for i in range(num_tries):
```

Comment:
Just a small change to avoid shadowing the `i` variable defined above.
```suggestion
                for attempt in range(num_tries):
```

Reply:
oof yeah, good catch!